### PR TITLE
Issue 4215 - Retire autocomplete passive scan rule

### PR DESCRIPTION
**Pull request by kingthorin**
PasswordAutocompleteScanner.java > Removed
PasswordAutocompleteScannerUnitTest.java > Removed
ZapAddOn.xml > Incremented version, scanner list, and changes section.
pascanrules.html > Removed related content.
Messages.properties > Removed related entries.

See zaproxy/zaproxy#4215

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscanrules/ZapAddOn.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-	<version>21</version>
+	<version>22</version>
 	<status>release</status>
 	<description>The release quality Passive Scanner rules</description>
 	<author>ZAP Dev Team</author>
 	<url/>
 	<changes>
 	<![CDATA[
-	Update for 2.7.0, Minor code changes to address deprecation.<br>
+	Retired the password auto-complete passive scan rule 9Issue 4215).<br>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrules/ZapAddOn.xml]: There's a typo before the issue.

**Comment by thc202** [comment]
Thanks!

**Comment by kingthorin** [comment]
@psiinon this one ok? You approved the accompanying one yesterday (in the main repo) which actually had the fix tag and closed the issue 😀

**Comment by psiinon** [comment]
Yeah, sorry - missed this one ;)

**Comment by kingthorin** [comment]
No worries, thanks!

**Pull request merged**